
turtle:
  K_pose: 0.5
  ref_x: 3.0
  ref_y: 3.0
